src/GenresList/GenresList.tsx

```diff
@@ -1,7 +1,8 @@
 import React, { Component } from 'react'
 import { getGenres } from '../apiCalls'
 
-interface IProps {}
+// come back and properly type setAppGenre
+interface IProps {setAppGenre: (genre: string) => void}
```

Comment:
This is a really nice check, to ensure that function prop that we are passing down is accepting the correct parameter and return the way we specify.